x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/index.test.tsx

```diff
@@ -119,7 +119,7 @@ describe('Hosts Table', () => {
             .find('.euiTable thead tr th button')
             .at(1)
             .text()
-        ).toEqual('Last SeenClick to sort in ascending order');
+        ).toEqual('Last seen Click to sort in ascending order');
```

Comment:
Nice fix with the `iInCircle` choice over the question mark and adding the spaces.

I did a quick search and I think we have other spots in the code base that could use this same treatment with proper spacing:
<img width="638" alt="Screen Shot 2019-07-31 at 10 50 19 AM" src="https://user-images.githubusercontent.com/1151048/62231306-02c30880-b381-11e9-8987-565777a11547.png">

That would be:
* BytesClick
* Domain nameClick
* NameClick (hosts table)
* PacketsClick
*  NameClick (users table)

Reply:
Looking closer, some of those strings might be correct as they map to a draggable and in a test when you run a `.text()` on them, it will not add a space which is expected.

Reply:
Yeah, in this case, the space is being created between `Last seen` and `Click to sort in ascending order` due to the space between the text and the column header's tooltip icon. Probably going to have to leave this one as is.